Wrote an assignment jv-jdbc.

ASelivonchyk opened the pull request:
// injector not working, is it normal?
- created Manufacturer class as manufacturer model;
- created ManufacturerDao with CRUD methods and implement it in class ManufacturerDaoImpl;
- created ConnectionUtil class which establishes connection to DB;
- created custom exception - DataProcessingException;
- made init_db.sql with parameters to create DB and table;
- in Main class call all of ManufacturerDao methods;

ASelivonchyk committed:
Wrote an assignment jv-jdbc.
// injector not working, is it normal?
- created Manufacturer class as manufacturer model;
- created ManufacturerDao with CRUD methods and implement it in class ManufacturerDaoImpl;
- created ConnectionUtil class which establishes connection to DB;
- created custom exception - DataProcessingException;
- made init_db.sql with parameters to create DB and table;
- in Main class call all of ManufacturerDao methods;

MaliukDaria commented on src/main/java/mate/jdbc/Main.java:
```diff
@@ -1,7 +1,25 @@
+        manufacturerDao.getAll().forEach(System.out::println);
+        hyundai.setCountry("Korea");
+        manufacturerDao.update(hyundai);
+        System.out.println(manufacturerDao.get(3L));
```
```suggestion
        System.out.println(hundai.getId());
```

Better not to get attached to a certain id, we can get it from Manufacturer entity

MaliukDaria commented on src/main/java/mate/jdbc/Main.java:
```diff
@@ -1,7 +1,25 @@
+        hyundai.setCountry("Korea");
+        manufacturerDao.update(hyundai);
+        System.out.println(manufacturerDao.get(3L));
+        System.out.println(manufacturerDao.delete(1L));
```
same here

MaliukDaria commented on src/main/java/mate/jdbc/dao/impl/ManufacturerDaoImpl.java:
```diff
@@ -0,0 +1,106 @@
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        String insertRequest = "INSERT INTO manufacturers(name, country) values(?, ?);";
+        try (var connection = ConnectionUtil.getConnection();
+                var createManufacturerStatement = connection
+                        .prepareStatement(insertRequest, Statement.RETURN_GENERATED_KEYS)) {
```
I think we should not use `var` here(same for other methods)

ASelivonchyk committed:
- fixed exception message in delete method ManufacturerDaoImpl class;
- fixed return statement in update method ManufacturerDaoImpl class;
- changed variable "var" in all methods;
- in Main method changed "hardcode" id to id from Manufacturer entity;

MaliukDaria reviewed (approved):
Good job! 🚀
